certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+        "operationId": "FindMultipleDocuments",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "x-ms-summary": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/{{CONTENT_TYPE}}",
+            "description": "Content-Type"
+          },
+          {
+            "name": "Access-Control-Request-Headers",
+            "x-ms-summary": "Access-Control-Request-Headers",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*",
+            "description": "Access-Control-Request-Headers"
+          },
+          {
+            "name": "api-key",
+            "x-ms-summary": "api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "{{API_KEY}}",
+            "description": "api-key"
+          },
+          {
+            "name": "Accept",
+            "x-ms-summary": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/{{CONTENT_TYPE}}",
+            "description": "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dataSource": {
+                  "type": "string",
+                  "description": "datasource",
+                  "title": ""
+                },
+                "database": {
+                  "type": "string",
+                  "description": "database"
+                },
+                "collection": {
+                  "type": "string",
+                  "description": "collection"
+                },
+                "filter": {
+                  "type": "object",
+                  "properties": {
+                    "age": {
+                      "type": "object",
+                      "properties": {
+                        "$lt": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "$lt"
+                        }
+                      },
+                      "description": "age"
+                    }
+                  },
+                  "description": "filter"
+                }
+              },
+              "default": {
+                "dataSource": "{{DATA_SOURCE}}",
+                "database": "{{DATABASE}}",
+                "collection": "{{COLLECTION}}",
+                "filter": {
+                  "age": {
+                    "$lt": 40
+                  }
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A JSON object containing array of all the documents fetched",
```

Comment:
@DianaMDB make sure everything is grammatically correct, you can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
Have changed "containing array" to "containing an array".